examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -235,6 +236,21 @@ CHIP_ERROR PairingCommand::EnableNetwork()
     else
     {
         networkId = ByteSpan(kTemporaryThreadNetworkId, sizeof(kTemporaryThreadNetworkId));
+
+        // use extendedPanId if available
+
+        Thread::OperationalDataset dataset;
+        CHIP_ERROR err = dataset.Init(mOperationalDataset);
+        if (err == CHIP_NO_ERROR)
+        {
+            static uint8_t sExtendedPanId[Thread::kSizeExtendedPanId];
```

Comment:
usage of static here seems suspicious - seems necessary because lifetime, however could we get away by placing this as a top-level item or making it a member of the PairingCommand instead?

Having a static array somewhat deep inside a method is not ideal.

Reply:
Simplified the handling of Thread network id and merged the 2 variables into one. It would be nice if GetExtendedPanId() API gave us a ByteSpan result instead of filling in an external buffer, then we wouldn't even need one.